web/src/lightbox.ts

```diff
@@ -218,10 +220,9 @@ export function render_lightbox_media_list(displayed_source: string): void {
             ".focused-message-list .message_inline_image img, .focused-message-list .message_inline_video video",
         ).toArray();
         const $media_list = $("#lightbox_overlay .image-list").empty();
-
         for (const media of media_list) {
             const src = canonical_url_of_media(media);
-            const className = displayed_source === src ? "image selected" : "image";
+            const className = src.endsWith(displayed_source) ? "image selected" : "image";
```

Comment:
I think we should canonicalize the displayed source with respect to the current window location, so we can compare them exactly?

Reply:
Is there a good or convenient way to do that with `payload.source`?

Reply:
(That's what's ultimately passed in as `displayed_source`)

Reply:
I guess it can be done by `url = new URL(payload.source, window.location.href).href` ?

Reply:
I've pushed a version that does that, @roanster007. Thank you!